src/hooks/useDialog.ts

```diff
@@ -0,0 +1,20 @@
+import { atom, useAtom } from 'jotai'
+
+const dialogAtom = atom(false)
+
+export function useDialog(){
+  const [isOpen, setIsOpen] = useAtom(dialogAtom)
```

Comment:
Use setIsOpen directly, no need for other methods

Reply:
got it